File: 01/RectangleArea.java
/* CircleArea.java computes the area of a circle.
 *
 * Input: The length and width of a rectangle.
 * Precondition: The length and width are positive numbers.
 * Output: The area of the a rectangle
 *
 * Author: Brian Goins (bmg23)
 * Date:02/05/2018
 **********************************************************/
import java.util.Scanner;

public class RectangleArea {

	//Function to calc rectangle's area.
	public static double rectArea(double l, double w) {
		return l * w; 

	}//Rectangle Area Method


	//Main Program
	public static void main(String[] args) {
		//Prompt for length and width. 
		System.out.println("\n\nTo compute the area of a rectangle,");
		System.out.print("enter the length: ");
		
		//Create a connection to the keyboard.
		Scanner keyboard = new Scanner(System.in); 

		double length = keyboard.nextDouble();
		
		System.out.print("\nand enter the width:");
		
		double width = keyboard.nextDouble();

		
      		// output area
      		System.out.println("\nThe area is " + rectArea(length, width) + "\n");
	}//Main Method
}//class RectangleArea

File: 01/rectangle_area.rb
#! /urs/bin/ruby
# rectangle_area.rb computes the area of the rectangle. 
# Input: the length and width of the rectangle
# Precodition: the length and width are positive. 
# Author: Brian Goins (bmg23)
# Date: 02/05/2019
#######################################################

def rectArea(length, width)
    length * width
end

if __FILE__ == $0
    puts "To compute the area of a rectangle,"
    print "enter the length: "
    length = gets.chomp.to_f
    print "enter the width"
    width = gets.chomp.to_f
    print "The area of the rectangle is: "
    puts rectArea(length, width)
end
